src/Components/sendSms/send_sms.js

```diff
@@ -0,0 +1,46 @@
+require('dotenv').config();
+
+const twilio = require('twilio');
+const client = new twilio(process.env.TWILIO_ACCOUNT_SID, process.env.TWILIO_AUTH_TOKEN);
+
+  client.messages
+      .create({
+        body: `{user.firstName + user.lastName} has started their trip. Please be on the look out for a message. You will be notified once their trip is complete, or if you need to contact them.`,
+        from: '+18284758343',
+        to: '+17083630654',
+      }, function (err, message) {
+        if (err) {
+          console.error('Text failed because: ' + err.message);
```

Comment:
Well done figuring out how this all works... looking forward to digging into it further.